docs/reference/rest_api_communities.md

```diff
@@ -19,7 +19,9 @@
 |-----------------|--------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
`### Community access` -> `#### Community access` on L16 since it's a sub-level .